go.mod

```diff
@@ -104,6 +104,7 @@ require (
 	go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin v0.49.0
 	go.opentelemetry.io/otel v1.31.0
 	go.opentelemetry.io/otel/metric v1.31.0
+	go.opentelemetry.io/otel/sdk/metric v1.31.0
```

Comment:
this didn't come with a corresponding go.sum change?

Reply:
No because it's still included, just sorted differently.